components/withBreadcrumbs.tsx

```diff
@@ -20,11 +21,55 @@ const WithBreadcrumbs: FC = () => {
       return [];
     }
 
-    return getSideNavigation([navigationKey as NavigationKeys])
-      .map(([, item]) => item.items)
-      .flat()
-      .filter(([, item]) => pathname.includes(item.link))
-      .map(([, item]) => ({ label: item.label, href: item.link }));
+    const navigationTree: Array<[string, MappedNavigationEntry]> =
+      getSideNavigation([navigationKey as NavigationKeys]);
+
+    const toCamelCase = (str: string) => {
+      return str
+        .split('-')
+        .map((word, index) =>
```

Comment:
I assume you're capitalising the string here. You don't need to do this with JavaScript. CSS can do that. Also we do have an utility for camel case already in the code I assume, (check util folder)

Reply:
There is no such fucniton util. Should I move it to util as a new file?

Reply:
Yeah that's what Im saying, I even gave a snippet that should be better (more generic)